### PR TITLE
Support installation to system directories. Fix errors with python3.

### PR DESCRIPTION
**Pull request by ildus**

### CODE REVIEW

File: duel/expr.py [hunk 1]
```diff
@@ -172,7 +172,7 @@ def __init__(self, n, a1, to):
         self.name_, self.to = n, to
     def eval(self):
         for n1,v1 in self.arg1_.eval():
-            for i in xrange(0 if self.to else v1, v1 if self.to else sys.maxint):
+            for i in range(0 if self.to else v1, v1 if self.to else sys.maxint):
                 v = gdb.Value(i).cast(v1.type)
```
Comment (vuvova) [inline duel/expr.py]: 1. I believe that python3 doesn't have `sys.maxint`, I'll replace it with an explicit constant 
2. I cannot use `range` in python2, `range(0, sys.maxint)` will try to create a list that long and will run out of memory. I'll use the same workaround I have in `pretty_printer.py`:
```python
try: a=xrange(0,10)
except: xrange=range
```

Comment (ildus) [reply duel/expr.py]: Maybe you should use `six` for range. It would be a nicer solution.

**Comment by ildus** [comment]
I've added a fix for sys.maxint

**Comment by vuvova** [comment]
1. Can six be considered almost universally installed everywhere? I don't want to add a new dependency when a couple of lines (`try`/`except`) will do. I'd rather switch to six later, when I'll be really needed.
2. Why would one use `six.MAXSIZE` at all, if `sys.maxsize` is present in both in python2 and python3?
3. you've done a bunch of changes that I don't understand, moving imports around, replacing `import help` with `import duel.help`, and using `os.path.join`. Why is that?

I tried a minimal patch (see https://github.com/vuvova/gdb-tools/commit/4f850c83976c480c71a45b86b42a27841265abf9) and it _looked_ like it was sufficient. I could install the package just fine and it worked after that. What am I missing?

**Comment by ildus** [comment]
1 and 2 depends on you. `six` is very common thing.
3. I mostly changed imports because they don't work the way you specified them in python3. Look at: https://stackoverflow.com/questions/12172791/changes-in-import-statement-python3

**Comment by ildus** [comment]
After all when you publish this repository to PyPI (if you publish it), you can just add `six` to dependencies, it will be installed like normal dependency.

**Comment by ildus** [comment]
Forgot to explain os.path.join, pyc files will be in `__pycache__` in python3. Also they can be turned off. You can't rely on `pyc` files to do something.

**Comment by vuvova** [comment]
imports - got it, thanks.
PyPi - yes, I'll publish it. Thanks a lot for the help with it.
os.path.join - I used `rstrip` meaning, to remove any of the characters 'p', 'y', 'c'. So it works both for .py and .pyc files, and (I've just tested) works with `__pycache__` too.

**Comment by vuvova** [comment]
Fixed, pushed. Thanks!

**Pull request closed without merging**